app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,6 @@
+class DashboardController < ApplicationController
+
+	def index
```

Comment:
So I recently switched to sublime. I have my tabs set to two tabs, and in my text editor they are at two spaces, but sometimes in my github they show up as 4 spaces. It's really weird. @case-eee